third_party/blurmac/Cargo.toml

```
@@ -7,6 +7,7 @@ keywords = ["bluetooth", "ble", "macOS", "CoreBluetooth"]
 repository = "https://github.com/akosthekiss/blurmac"
 authors = ["Akos Kiss <akiss@inf.u-szeged.hu>"]
 license = "BSD-3-Clause"
+edition = "2015"
```

```suggestion
edition = "2018"
```
The version we vendored included an update to 2018.